lib/eightball.rb

```diff
@@ -1,12 +1,11 @@
 # encoding: utf-8
 
 module Eightball
-
   def eightball(replacement_string = '?')
-    string = self
```

Comment:
Why did this line get removed?  I'd be concerned that things don't work with this line removed.

Reply:
Rubocop suggested it was redundant